exchanges/gateio/gateio_wrapper.go

```diff
@@ -306,31 +663,52 @@ func (g *Gateio) FetchOrderbook(ctx context.Context, p currency.Pair, assetType
 }
 
 // UpdateOrderbook updates and returns the orderbook for a currency pair
-func (g *Gateio) UpdateOrderbook(ctx context.Context, p currency.Pair, assetType asset.Item) (*orderbook.Base, error) {
-	book := &orderbook.Base{
-		Exchange:        g.Name,
-		Pair:            p,
-		Asset:           assetType,
-		VerifyOrderbook: g.CanVerifyOrderbook,
-	}
-	curr, err := g.FormatExchangeCurrency(p, assetType)
+func (g *Gateio) UpdateOrderbook(ctx context.Context, p currency.Pair, a asset.Item) (*orderbook.Base, error) {
```

Comment:
This is for `Margin` and pair `BCHA_USDT`:
```
[ERROR] | ORDERBOOK | 11/04/2023 10:56:50 | Failed to get REST orderbook. Error: GateIO unsuccessful HTTP status code: 400 raw response: {"label":"INVALID_CURRENCY_PAIR","message":"Invalid currency pair BCHA_USDT"}
```

Reply:
I have fetched orderbook information for margin and cross-margin instruments, which has an orderbook data of the spot. Meaning, Margin, and Cross margin instruments have a Funding book, not an order book.

Reply:
That is not an OK answer. GCT is returning errors for attempting to get a rest orderbook and tickers. This occurs after timing out the websocket to retrieve this data. If the above currency, which is added to the Margin asset type, is not a valid currency, it shouldn't be being processed.

If the alternative is that margin etc pairs do not have tickers and orderbooks, then that needs to be handled instead of impacting the websocket and returning errors in the REST response.